src/templates/kit/screens/DotVotingScreen.js

```diff
@@ -0,0 +1,342 @@
+import React, { useCallback, useReducer, useState, useRef } from 'react'
+import PropTypes from 'prop-types'
+import { GU, Help, Info } from '@aragon/ui'
+import {
+  Duration,
+  Header,
+  KnownAppBadge,
+  Navigation,
+  PercentageField,
+  ScreenPropsType,
+} from '..'
+
+const MINUTE_IN_SECONDS = 60
+const HOUR_IN_SECONDS = MINUTE_IN_SECONDS * 60
+const DAY_IN_SECONDS = HOUR_IN_SECONDS * 24
+
+const DEFAULT_SUPPORT = 0
+const DEFAULT_QUORUM = 15
+const DEFAULT_DURATION = MINUTE_IN_SECONDS
+const DEFAULT_DISCUSSIONS = false
+
+function validationError(duration) {
+  if (duration < 1 * MINUTE_IN_SECONDS) {
+    return 'Please ensure the vote duration is equal to or longer than 1 minute.'
+  }
+  return null
+}
+
+function reduceFields(fields, [field, value]) {
```

Comment:
Would something like this work for you here?

```JavaScript
return { ...fields, ...(['discussions', 'duration', 'quorum', 'support'].includes(field) ? { [field]: value } : {}) }
```

Reply:
Makes sense, it was a copy-pasta from the VotingScreen where I recall the same refactor was there, thanks for pointing out @AquiGorka !